src/objects.rs

```diff
@@ -18,29 +18,38 @@ use std::ops::{Deref, DerefMut};
 /// information.
 #[derive(Debug, Default, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
 pub struct Tags(TagsImpl);
+
 /// FlatMap representing the key-value pairs of the tags
 pub type TagsImpl = ::flat_map::FlatMap<String, String>;
 impl Tags {
     /// Creates a new, empty `Tags` object.
     pub fn new() -> Tags {
         Tags(TagsImpl::new())
     }
-    /// Returns if contains the tag `(key, value)`.
-    pub fn contains(&self, key: &str, value: &str) -> bool {
-        self.0.get(key).map_or(false, |v| v.as_str() == value)
+    /// Returns the key associated value or `None` if there is no such tag.
+    pub fn get(&self, key: &str) -> Option<&str> {
+        self.0.get(key).map(|v| v.as_str())
+    }
+    /// Returns `true` if contains a tag with the given `key`.
+    pub fn contains(&self, key: &str) -> bool {
+        self.0.get(key).is_some()
```

Comment:
No, because:
 - that's a breaking change
 - that's not the idiomatic API: you can already do `tags.contains_key("foo")`

This `contains` method is here because that's something you want a lot, and is easier to write than your `get` variant. It follows the `Set` interface.

Reply:
> that's a breaking change

I changed the medium version of the crate.

> that's not the idiomatic API: you can already do `tags.contains_key("foo")`

That's a very good point, it should indeed be `contains_key`. Documentation failed me.

> This `contains` method is here because that's something you want a lot, and is easier to write than your `get` variant. It follows the `Set` interface.

I don't agree with this assumption: having a `get` is more idiomatic. `contains` is misleading in my opinion (that's mainly why I didn't think of `contains_key`).

Reply:
> that's a breaking change

I changed the medium version of the crate.

> that's not the idiomatic API: you can already do `tags.contains_key("foo")`

That's a very good point, it should indeed be `contains_key`. Documentation failed me.

> This `contains` method is here because that's something you want a lot, and is easier to write than your `get` variant. It follows the `Set` interface.

I don't agree with this assumption: having a `get` is more idiomatic. `contains` is misleading in my opinion (that's mainly why I didn't think of `contains_key`).

Reply:
You already have `get` and `contains_key` via `Deref`

Reply:
Then this method shouldn't be there.